include/emulator.hpp

```diff
@@ -50,6 +51,7 @@ class Emulator {
 	Crypto::AESEngine aesEngine;
 	MiniAudioDevice audioDevice;
 	Cheats cheats;
+	Mappings keyboardMappings;
```

Comment:
(this should probably be part of the frontend classes since the emulator core itself doesn't use them and they're frontend-specific)